src/main/kotlin/org/openbroker/se/privateunsecuredloan/events/OfferAccepted.kt

```diff
@@ -15,7 +15,8 @@ data class OfferAccepted @JvmOverloads constructor(
     val ssn: String? = null,
     val ssnCoapplicant: String? = null,
     val emailAddress: String? = null,
-    val emailAddressCoapplicant: String? = null
+    val emailAddressCoapplicant: String? = null,
+    val extensions: Map<String, Any> = emptyMap()
```

Comment:
I'm not sure. This should never happen

Reply:
But maybe you are right, in case when something goes wrong on communication in db layer in our implementation and make it more flexible, it is better to make it nullable.